app/assets/stylesheets/layouts/_header.css.scss

```diff
@@ -1,4 +1,28 @@
-.header {
-  &.top-bar { background: $header-green !important; }
+nav.header {
+  background: $header-green !important;
   li, a { background: $header-green !important; }
 }
+
+.magellan.header {
+  background: $header-green !important;
+
+  ul, li { margin: 0; }
+  li, a { background: $header-green !important; }
+
+  a {
+    color: white !important;
+    &:hover { color: $bright-green; }
+    padding: 0;
+  }
+
+  dd.active {
+    background: white !important;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used